app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -103,6 +103,10 @@ export const ScanningQRCodeScreen: React.FC = () => {
     }
   }, [hasPermission, requestPermission])
 
+  const loadInBrowser = (url: string) => {
+    Linking.openURL(url).catch((err) => console.error("Couldn't load page", err))
```

Comment:
Should probably be showing an error message to the user here instead of using the console

Reply:
Put in an alert